ansible_runner/runner_config.py

```diff
@@ -104,6 +105,9 @@ def __init__(self,
         self.process_isolation_hide_paths = process_isolation_hide_paths
         self.process_isolation_show_paths = process_isolation_show_paths
         self.process_isolation_ro_paths = process_isolation_ro_paths
+        self.resource_profiling = resource_profiling
```

Comment:
These values also need to be able to come in from settings:

https://github.com/ansible/ansible-runner/blob/c70a02670882d94ae189a32b77a5f07df205dab9/ansible_runner/runner_config.py#L251

Since these values also can come in from the module interface they also need to be documented here:

https://github.com/ansible/ansible-runner/blob/c70a02670882d94ae189a32b77a5f07df205dab9/ansible_runner/interface.py#L116

(along with supporting `type` information)